src/index.js

```diff
@@ -37,7 +37,31 @@ const useLocalStorage = (key, {updateFrequency}) => {
     }
   }, [])
 
-  return [value, writeToLocalStorage]
+  return [value, write]
 }
 
-export default useLocalStorage
+const useLocalStorageNoSync = key => {
+  const [value, setValue] = useState()
+
+  const read = () => {
+    const oldValue = value
+    const newValue = localStorage.getItem(key)
+    if (newValue !== oldValue) {
+      setValue(newValue)
+    }
+  }
+
+  const write = newValue => {
+    if (newValue !== undefined) {
+      localStorage.setItem(key, newValue)
+    }
+    else {
+      localStorage.removeItem(key)
+    }
+    setValue(newValue)
+  }
+
+  return [read, write]
+}
+
+export default {useLocalStorage, useLocalStorageNoSync}
```

Comment:
Missing new line EOF

Reply:
Added in 20f4b58108e927ae0b84166d13eca28bf6149ce5